packages/terra-content-container/src/ContentContainer.jsx

```diff
@@ -54,7 +54,7 @@ const ContentContainer = ({
     <div {...customProps} className={contentLayoutClassNames}>
       {header && <div className={cx('header')}>{header}</div>}
       <div className={cx('main')}>
-        <Scroll className={cx('normalizer')} refCallback={scrollRefCallback}>
+        <Scroll tabIndex={fill ? 0 : -1} className={cx('normalizer')} refCallback={scrollRefCallback}>
```

Comment:
When clicking on this object in the standard container, the outline shows up.  `Scroll` needs to have `outline: none` assigned to it.

Reply:
We aren't going to be able to do this broadly. Multiple nested content container will have major issues with tabindexes and break accessibility. This needs evaluation at the framework level. For the time being we may need to find a way to alter the config to turn off this axe rule until such time as we can circle back to this.